app/controllers/provider/sessions_controller.rb

```diff
@@ -28,12 +28,12 @@ def create
       create_user_session!(strategy&.authentication_provider_id)
       flash[:notice] = 'Signed in successfully'
 
-      AuditLogService.call("Signed in: #{current_user.id}/#{current_user.username} #{current_user.first_name} #{current_user.last_name}")
-
       redirect_back_or_default provider_admin_path
     else
       new
       flash.now[:error] ||= strategy&.error_message
+      attempted_cred = auth_params.fetch(:username, 'SSO')
```

Comment:
Why the fallback is "SSO"? :thinking:

Reply:
hm, I am not quite sure about the SSO flow, I'd need to check, but would there potentially be some value in logging the `system_name` of the auth provider instead?... just thinking out loud.

Reply:
Maybe, that would allow admins to know which auth provider was being used when it failed to log in. Probably that's better than just print `SSO`